public/rest.js

```diff
@@ -1,38 +0,0 @@
-//this gathers the name of the restaurant from the URL and stores it in a variable
-let restName = document.location.pathname.split('/').splice(-1)
-
-//the variable is then used in a fetch that should gather information from the .json file attached to it
-fetch(`/restaurant/${restName}`)
-    //when the info is gathered,
-    .then((res) => res.json())
-    //execute the fillEntry function, which will fill in info on individual HTML webpage
-    .then(fillEntry)
```

Comment:
Either you are calling this function incorrectly or should be parsing the data within the fetch, which would make it easier